portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,13 +37,28 @@
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> comments = new ArrayList<>();
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
-    PreparedQuery results = this.datastore.prepare(query);
-    for (Entity entity : results.asIterable()) {
-      comments.add((String) entity.getProperty("content"));
+    int limit = 3;
```

Comment:
is this value (3) ever used?

Reply:
Not really -- unless something goes wrong and the default limit (3) set in HTML won't come through. Is this too extra?

Reply:
Ok I'm convinced and the initial value is now removed.